package.json

```diff
@@ -56,6 +58,7 @@
     "lunr": "^2.3.9",
     "marked": "^1.2.7",
     "node-fetch": "^2.6.1",
+    "node-sass": "^5.0.0",
```

Comment:
This should technically be a dev dependency (I know that other dependencies aren't specified properly, but we can fix that another time). Maybe we should just fix that in bulk in a followup at this point though, you can leave this for now.